src/homework/lesson_6/MainClient.java

```diff
@@ -0,0 +1,56 @@
+package homework.lesson_6;
+
+import java.util.Scanner;
+
+public class MainClient {
+    public static void main(String[] args) {
+        // Тестирование клиента
+        try {
+            new ChatClient(new IChatViewModel() {
+                @Override
+                public void onNewMessage(Message message) {
+                    System.out.println(message.toString());
+                }
+
+                @Override
+                public String getMessageText() {
+                    return scanner.nextLine();
+                }
+
+                @Override
+                public String getHost() {
+                    return "localhost";
+                }
+
+                @Override
+                public int getPort() {
+                    return 8081;
+                }
+
+                @Override
+                public String getNickName() {
+                    return "ititov";
+                }
+
+                @Override
+                public void onStart() {
+                    scanner = new Scanner(System.in);
+                }
+
+                @Override
+                public void onClose() {
+                    System.exit(0);
+                }
+
+                @Override
+                public void onSetLog(String message) {
+                    System.out.println(message);
+                }
+            }).Start();
```

Comment:
I detect that this code is problematic. According to the [Bad practice (BAD_PRACTICE)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#bad-practice-bad-practice), [Nm: Method names should start with a lower case letter (NM_METHOD_NAMING_CONVENTION)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#nm-method-names-should-start-with-a-lower-case-letter-nm-method-naming-convention).
Methods should be verbs, in mixed case with the first letter lowercase, with the first letter of each internal word capitalized.